src/app/pages/errata-summary/table/table.js

```diff
@@ -62,81 +62,201 @@ const sortFunctions = {
     },
     sortNumber: (a, b) => a - b,
     sortDecision: (a, b) => {
-        /* eslint complexity: 0 */
-        const ar = statusSortOrder[a.substr(0, 2)];
-        const br = statusSortOrder[b.substr(0, 2)];
+        const ar = statusSortOrder[a.substr(0, 2)] || 6;
+        const br = statusSortOrder[b.substr(0, 2)] || 6;
 
         if (ar !== br) {
             return br - ar;
         }
-        const ad = new Date(a.modified);
-        const bd = new Date(b.modified);
+        const ad = new Date(a.modified).valueOf();
+        const bd = new Date(b.modified).valueOf();
 
-        if (ad < bd) {
-            return -1;
-        }
-        return 1;
+        return bd - ad;
     }
 };
 
-const spec = {
-    template,
-    view: {
-        classes: ['boxed']
-    },
-    model() {
-        return {
-            columnSpecs,
-            errataEntries: this.errataEntries(),
-            sortDir: this.sortDir,
-            sortKey: this.sortKey
-        };
-    }
-};
+function DesktopHeaderColumn({md, sortController}) {
```

Comment:
I'm personally not a fan of really short variable names since it can make maintenance more difficult in the future. I'm not able to sort out what md is from the code. Should it have a longer name?